src/controllers/accounts/AccountsStakingPayoutsController.ts

```diff
@@ -104,17 +104,18 @@ export default class AccountsStakingPayoutsController extends AbstractController
 		const earlyErasBlockInfo: IEarlyErasBlockInfo = kusamaEarlyErasBlockInfo;
 		let hash = await this.getHashFromAt(at);
 		let apiAt = await this.api.at(hash);
+		const runtimeInfo = await this.api.rpc.state.getRuntimeVersion(hash);
 		const { eraArg, currentEra } = await this.getEraAndHash(apiAt, this.verifyAndCastOr('era', era, undefined));
-		if (currentEra <= 519 && depth !== undefined) {
+		if (currentEra <= 519 && depth !== undefined && runtimeInfo.specName.toString() === 'kusama') {
```

Comment:
Why is the 519 and the one below but the other check is 518?

Reply:
This is an edge case where the old logic changes to the current one, it's on purpose, otherwise you completely skip 518

Reply:
Ahh i see nice. Thanks for the explanation